sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/sweep/parameterized_sweep.py

```diff
@@ -89,18 +91,30 @@ def __init__(
         ~azure.ai.ml.sweep.Randint, ~azure.ai.ml.sweep.Uniform]]
         :param queue_settings: Queue settings for sweep job.
         :type queue_settings: ~azure.ai.ml.entities.QueueSettings
+        :param resources: Compute Resource configuration for the job.
+        :type resources: ~azure.ai.ml.entities.ResourceConfiguration
         """
         self.sampling_algorithm = sampling_algorithm
         self.early_termination = early_termination
         self._limits = limits
         self.search_space = search_space
         self.queue_settings = queue_settings
+        self.resources = resources
 
         if isinstance(objective, Dict):
             self.objective = Objective(**objective)
         else:
             self.objective = objective
 
+    @property
+    def resources(self) -> JobResourceConfiguration:
+        """Resources for sweep job.
+
+        :returns: Resources for sweep job.
+        :rtype: ~azure.ai.ml.entities.ResourceConfiguration
+        """
+        return self._resources
```

Comment:
would it be better if we give resources a default value in getter so that customer can use below code to update resource configuration?
```python
sweep.resources.xxx = xx
```

Reply:
i was finding this as more cleaner way, may be we can consider it for follow up PR ?

Reply:
sure.